src/main/resources/i18n/messages.properties

```diff
@@ -681,4 +681,9 @@ query.notContains=not contains
 game.couldNotJoin=Could not join game with id id ''{0}''.
 discord.disallowJoins=Disallow players joining via Discord
 replay.couldNotOpen=Replay with id ''{0}'' , could not be opened.
-error.game.notTerminatedCorrectly=The game did not terminate correctly and has still certain files open. Some files from the installed game could not be copied over to FAF but unless you recently modified game files in the installed version you probably do not care. We are still starting the game for you.
\ No newline at end of file
+error.game.notTerminatedCorrectly=The game did not terminate correctly and has still certain files open. Some files from the installed game could not be copied over to FAF but unless you recently modified game files in the installed version you probably do not care. We are still starting the game for you.
+revert.alreadySymlinked=Revert this I got it symlinked
```

Comment:
Is this meant to be shown?

Reply:
The user has the option not to create a symlink in case he already did it manually.

Reply:
Well if you have a symlink and we change the location you see everything twice